### DISCUSSION TITLE
Wifi in Access Point mode crashes silenty

### DISCUSSION THREAD

**Discussion by Tobinator32** [micropython/micropython#12754] [ESP32]
Im currently developing a web server based application using MicroWebSrv2. Although its running great and i've been able to resolve most of the crashes by thinning out the TCP backlog of the server (setting it to 2) sometimes the Wifi stack crashes without any notice. It happens when reloading a page and pressing the back button before it completly finished loading the current page. As it has to be bullet proof is there any advice how to mitigate or work around any issues related with that? How can i detect if the AP goes down in the first place. This should be a last resort so that i am able to restart wifi.
Kind Regard Tobi

**Comment by jimmo** [comment]
@Tobinator32 Can you give more detail on what you mean by "the wifi stack crashes". Thanks

**Reply by Tobinator32** [reply]
I can't no longer login with a new device as it says connection unsuccessful or it disappears completely. All of this happens immediately after it crashes on the client already connected to the webserver and jumping sites/loading content.
